msom/src/main/java/pl/edu/uj/fais/wpz/msom/web/ModuleController.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package pl.edu.uj.fais.wpz.msom.web;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import pl.edu.uj.fais.wpz.msom.entities.Module;
+import pl.edu.uj.fais.wpz.msom.entities.TaskType;
+import pl.edu.uj.fais.wpz.msom.service.interfaces.ModuleService;
+import pl.edu.uj.fais.wpz.msom.service.interfaces.TaskTypeService;
+
+/**
+ *
+ * @author paweldylag
+ */
+@Controller
+public class ModuleController {
```

Comment:
Żeby poprawnie rozwiązywało siężkę, nazwa klasy controlera powinna nazywać się "ModulesController" (l.mnoga), czyli tak samo jak podstawowa wartość dla RequestMapping(value = "/modules"). Teoretycznie chyba można to jakoś ustawić, ale tak jest domyślnie.

Zastanawiam się też czy "Module" jako nazwa dla modułów przetwarzających jest odpowiednia - trochę się może mylić z modułami, to samo tyczy się kontrolerów.